solutions/standard/outputs.tf

```diff
@@ -58,3 +58,9 @@ output "service_credential_secret_groups" {
   description = "Service credential secret groups"
   value       = length(local.service_credential_secrets) > 0 ? module.secrets_manager_service_credentials[0].secret_groups : null
 }
+
+output "admin_pass" {
+  description = "Elasticsearch administrator password"
+  value       = local.admin_pass
+  sensitive   = true
+}
```

Comment:
We will need to output the secret name, secret ID, and secret group details if we are going to now create these for admin password

Reply:
as a part of `service_credential_secrets` output we output `admin_pass secret` value

```
service_credential_secrets = {
  "test-313-cred-reader" = {
    "secret_crn" = "crn:v1:bluemix:public:secrets-manager:us-south:a/abac0df06b644a9cabc6e44f55b3880e:c6698c1b-48fa-4d46-b7a9-90295fe2ca01:secret:9dc5296b-d1f0-0956-b050-5766791dc9f5"
    "secret_id" = "9dc5296b-d1f0-0956-b050-5766791dc9f5"
    "secret_next_rotation_date" = "2024-12-22T14:56:05Z"
    "secret_rotation" = true
    "secret_rotation_interval" = "89 day(s)"
  }
  "test-313-cred-writer" = {
    "secret_crn" = "crn:v1:bluemix:public:secrets-manager:us-south:a/abac0df06b644a9cabc6e44f55b3880e:c6698c1b-48fa-4d46-b7a9-90295fe2ca01:secret:26e38184-4fdb-bd89-0410-3d2c85f4cf4e"
    "secret_id" = "26e38184-4fdb-bd89-0410-3d2c85f4cf4e"
    "secret_next_rotation_date" = "2024-12-22T14:56:05Z"
    "secret_rotation" = true
    "secret_rotation_interval" = "89 day(s)"
  }
  "test-andrej-3-elasticsearch-administrator-secret2" = {
    "secret_crn" = "crn:v1:bluemix:public:secrets-manager:us-south:a/abac0df06b644a9cabc6e44f55b3880e:c6698c1b-48fa-4d46-b7a9-90295fe2ca01:secret:d17546be-26ba-5d7a-8a7f-2568a3f8aec7"
    "secret_id" = "d17546be-26ba-5d7a-8a7f-2568a3f8aec7"
    "secret_next_rotation_date" = null
    "secret_rotation" = tobool(null)
    "secret_rotation_interval" = "89 day(s)"
  }
}

```
maybe the output name should be changed to something more generic?

another output is `admin_pass = <sensitive>` which contains the value.
```
terraform output admin_pass
"NO0xxxx....xxx1Z"
```

Reply:
ok I see the confusion, the module `secrets_manager_service_credentials` is creating both the service credentials and the arbitrary admin pass secret.

The problem is the `service_credential_secrets` output will now also output the admin password even though its not a service credentials. Can we maybe add a new output for that?

Reply:
or maybe we just re-use `service_credential_secrets` and `service_credential_secret_groups` but rename them? Also not sure they these are separate outputs?